### PR TITLE
Process entity references as input to Multi-Select Screen

### PR DESCRIPTION
**Pull request by shubham1g5**
These changes are prompted by the [CLI support for Multi Select screen](https://dimagi-dev.atlassian.net/browse/USH-2012).

### CODE REVIEW

**Comment by shubham1g5** [comment]
looking into test failure...

File: src/cli/java/org/commcare/util/screen/EntityScreen.java [hunk 1]
```diff
@@ -241,7 +241,27 @@ protected boolean executePendingAction(CommCareSession session) {
...
+
+    /**
+     * Updates entity selected on the screen
+     *
+     * @param input          input to the entity selected on the screen
+     * @param selectedRefs   references for the selected entity, only contains a single element for the
+     *                       single-select entity screen
+     * @throws CommCareSessionException
+     */
+    public void updateSelection(String input, @Nullable TreeReference[] selectedRefs) throws CommCareSessionException {
```
Comment (snopoke) [inline src/cli/java/org/commcare/util/screen/EntityScreen.java:255]: is `selectedRefs` still nullable?

File: src/cli/java/org/commcare/util/screen/EntityScreen.java [hunk 2]
```diff
@@ -241,7 +241,27 @@ protected boolean executePendingAction(CommCareSession session) {
      */
     public void updateSelection(String input, @Nullable String[] selectedValues) throws CommCareSessionException {
```
Comment (snopoke) [inline src/cli/java/org/commcare/util/screen/EntityScreen.java:242]: is this ever called now?

**Review by snopoke — APPROVED** [review]

File: src/cli/java/org/commcare/util/screen/EntityScreen.java (hunk 1 above)
Comment (shubham1g5) [reply src/cli/java/org/commcare/util/screen/EntityScreen.java]: no, https://github.com/dimagi/commcare-core/pull/1117/commits/6439bf6568e603226fd9ce776ec26494420a468e

File: src/cli/java/org/commcare/util/screen/EntityScreen.java (hunk 2 above)
Comment (shubham1g5) [reply src/cli/java/org/commcare/util/screen/EntityScreen.java:242]: yes, this is still called for [normal FP navigation](https://github.com/dimagi/commcare-core/blob/entityScreenRefactor/src/cli/java/org/commcare/util/screen/EntityListSubscreen.java#L248), while the new updateSelection with TreeReferences is invoked for CLI only.

**Review by snopoke — APPROVED** [review]

**Pull request merged**